cypress.config.js

```diff
@@ -1,8 +1,34 @@
 const { defineConfig } = require('cypress')
+const faker = require('@faker-js/faker');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com/automation-practice-form',
+    viewportHeight: 1200,
+    viewportWidth: 1680,
     setupNodeEvents(on, config) {
+      on("task",{
+        generateUser() {
+          genders = ['Female', 'Male', 'Other'];
+          subjects = ['English', 'Chemistry', 'Computer Science'];
+          hobbies = ['Music', 'Sports', 'Reading'];
```

Comment:
Declare the arrays 'genders', 'subjects', and 'hobbies' as constants outside of the 'generateUser' function to avoid redeclaring them each time the function runs.